about.html

```diff
@@ -0,0 +1,157 @@
+                    My Page
+                </a>
+                <a href="#">
+                    Logout
+                </a>
+            </nav>
+            
+
+            <div class="burger">
+                <i class="fa fa-bars fa-3x" aria-hidden="true"></i>
+            </div>
+            <nav class="menu-mobile">
+                <ul class="nav-links-mobile">
+                    <li>
+                        <a href="index.html" class="a-links">Home</a>
+                    </li>
+                    <li>
+                        <a href="#venue" class="a-links">Schedule</a>
+                    </li>
+                    <li>
+                        <a href="#HOF" class="a-links">Speakers</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Partner</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Join Us!</a>
+                    </li>
+                    <li>
+                        <div class="close-icon">
+                            <i class="fa fa-times fa-3x" aria-hidden="true"></i>
+                        </div>
+                    </li>
+                </ul>
+            </nav>
+
+            <nav class="menu-bigscreen">
+                <a href="index.html" class="logo">
+                    <img src="assets/apple-touch-icon.png" alt="logo">
+                </a>
+                <ul>
+                    <li>
+                        <a href="index.html" class="a-links">Home</a>
+                    </li>
+                    <li>
+                        <a href="#venue" class="a-links">Schedule</a>
+                    </li>
+                    <li>
+                        <a href="#HOF" class="a-links">Speakers</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Partner</a>
+                    </li>
+                    <li>
+                        <a href="#prospects" class="a-links">Join Us!</a>
+                    </li>
+                </ul>
+
+            </nav>
+        </header>
+
+        <div id="abt-container">
+            <P class="p-3">Unleashing the Power of Sports in India</P>
+            <h1 class="head-1">Sport Summit <br> India 2023</h1>
+            <p class="box-2">
+                India has always had a rich history and culture in sports, from ancient games like Kabaddi and Kho Kho
+                to modern-day cricket and football. However, the potential of sports to transform the country goes
+                beyond just entertainment and recreation. The Sports Summit India aims to explore and harness the power
+                of sports to empower the nation in various ways. </p>
+           <P class="email-me">Please contact us per Email for any further questions about Sport Summit India 2023!</P>
+           <p class="email-id">ss2023@india.org</p>
+        </div>
```

Comment:
- [ ] Just like that of the `home` page, kindly use `main` tag for this section. and likewise.

- [ ] The design is overlapping, kindly fix it.
- [ ] It should have just one header, with the appropriate Behance design.
- [ ] The text in the white box, should also have a pixel perfect design appropriately.


- Your design
<img width="499" alt="image" src="https://user-images.githubusercontent.com/41163571/235315067-9e982a45-e2b7-4967-9e72-ea2dedff5079.png">

- Behance design
<img width="297" alt="image" src="https://user-images.githubusercontent.com/41163571/235315087-9d0f09d1-67c6-4e0b-9a7c-65a06d2ab1cd.png">